x/oracle/keeper/reward_unit_test.go

```diff
@@ -0,0 +1,24 @@
+package keeper
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/require"
+)
+
+func TestVoteTargetWithUmee(t *testing.T) {
+	require := require.New(t)
+	tcs := []struct {
+		in  []string
+		out []string
+	}{
+		{[]string{}, []string{"uumee"}},
+		{[]string{"a"}, []string{"uumee", "a"}},
+		{[]string{"x", "a", "heeeyyy"}, []string{"uumee", "x", "a", "heeeyyy"}},
+		{[]string{"x", "a", "uumee"}, []string{"x", "a", "uumee"}},
+	}
+	for i, tc := range tcs {
+		require.Equal(tc.out, voteTargetsWithUmee(tc.in), i)
+	}
```

Comment:
comments or named parameters would clarify what this test is doing

Reply:
@toteki isn't this clear? It checks if output of the function `voteTargetWithUmee` with `tc.in` as the input argument equals `tc.out`.

Reply:
Maybe a single comment between lines 14-15
```
// Should prepend "uumee" to a slice of denoms, unless it is already present.
```

Reply:
Ah, will add it.